### PR TITLE
Add back setup for 15SP1 quarterly updates

### PR DESCRIPTION
**Pull request by gyr**
Maintenance asked to add back 15SP1 quarterly updates because they have a request to update it

### CODE REVIEW

File: gocd/checkers.suse.gocd.yaml.erb [hunk 1]
```diff
@@ -281,6 +281,35 @@ pipelines:
             - staging-bot
             tasks:
             - script: ./legal-auto.py -A https://api.suse.de --debug --legaldb http://legaldb.suse.de project $(cat /home/go/config/legal-auto-projects-suse)
```
Comment (coolo) [inline gocd/checkers.suse.gocd.yaml.erb:283]: please program this as loop then

**Review by coolo — CHANGES_REQUESTED** [review]

File: gocd/checkers.suse.gocd.yaml.erb (hunk 1 above)
Comment (gyr) [reply gocd/checkers.suse.gocd.yaml.erb:283]: sorry, but did you mean by "loop"

Comment (coolo) [reply gocd/checkers.suse.gocd.yaml.erb:283]: Well, replace the copy&pasted code with a loop over SP1 and SP2. this is ruby

Comment (coolo) [reply gocd/checkers.suse.gocd.yaml.erb:283]: Take inspiration from 

https://github.com/openSUSE/openSUSE-release-tools/blob/6956f3b65580b41069282186c99ae254aa83bfe7/gocd/announcer.gocd.yaml.erb#L3

**Review by coolo — APPROVED** [review]

**Pull request merged**